src/test/specs/CloseNewTabSpec.ts

```diff
@@ -0,0 +1,44 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+chai.use(chaiAsPromised);
+import { BrowserUtils, Reporter } from '../..';
+import { describeCommon } from '../TestHelper';
+
+const OPEN_NEW_TAB: string = "//*[@data-test='open-new-tab']";
+
+/**
+ * wdio-allure-ts close tab action
+ */
+describeCommon('open new tab, and close it', () => {
+  it('click on `open new tab button` ', async () => {
+    await Reporter.debug('open new tab');
+    await BrowserUtils.click(OPEN_NEW_TAB);
+
+    await Reporter.debug('get tabs ids');
+    const tabIds: Array<string> = await BrowserUtils.getWindowHandles();
+
+    await Reporter.debug('validate number of tabs');
+    chai.assert.equal(tabIds.length, 2, 'Incorrect number of tab ids');
+  });
```

Comment:
this test is not relevant to the function `closeWindow()`, because in the test you don't check the function.